Monika After Story/game/script-topics.rpy

```diff
@@ -13478,3 +13478,67 @@ label monika_sweatercurse:
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_allegory_of_the_cave",
+            category=['philosophy'],
+            prompt="Seeing the real truth",
+            random=True
+        )
+    )
+
+label monika_allegory_of_the_cave:
+    m 6eua "So, I've been doing some reading on Plato lately."
+    if mas_getEV("monika_allegory_of_the_cave").shown_count == 0:
+        m 1etc "You've heard of him, right?{nw}"
+        $ _history_list.pop()
+        menu:
+            m "You've heard of him, right?{fast}"
+            "Yes.":
+                m 1wub "Really?"
+                m 3eua "I think that, outside of dedicated circles, not that many people know about him nowadays."
+            "No.":
+                m 1eua "It's not really a big deal, I can teach you about him."
+                m 3eua "Plato was a well known philosopher from Ancient Greece."
+        m 3eua "Anyway, something of his caught my eye recently.{w=0.5} The Allegory of the Cave."
+    else:
+        m 3eua "Specifically,{w=0.5} The Allegory of the Cave."
+
+    m 4eub "Imagine there're a bunch of people chained up, since childhood, in a cave and they can't look anywhere but straight ahead."
+    m 7eub "The wall in front of them were the shadows from people passing by, and the prisoners can hear these people's chatter reflected off the wall too."
+    m "All they know is shadows that make sounds. They'd really believe that the shadows were the ones making sounds!"
+    m "Wouldn't that be their reality? Even though it's just shades of grey to them, it's what makes up their entire idea of what the universe."
+    m 1ekb "Of course, it would be a bit difficult to open your eyes to the truth when you've believed a lie your whole life."
+    m "Now imagine that one of those prisoners were set free and forced out of the cave. His eyes would hurt so bad from the light he'd probably try to run straight back into the cave."
+    m 1esb "...Back to the real world, or his idea of it anyway."
+    m "But if you kept him out of the cave long enough, his eyes would adjust and eventually he'd start to understand colors."
+    m 2esb "And he'd realize that his old view of the world wasn't exactly an accurate representation of the truth. The truth is more than shadows on a wall."
+    m "If he ran back into the cave trying to tell the other prisoners about the truth, they'd probably see him as a madman and kill him if he tried to force them out."
+    m "After all, if he went back into the cave after his eyesight got used to the light then he'd pretty much be blind as a bat in there."
+    m "And if a blind guy came and tried to tell you that he's seeing a lot of colors, then you'd probaby think he's crazy too."
+    m 6esc "Anyway...does this remind you of anything?"
+    m 6dkc "..."
+    if persistent._mas_sensitive_mode:
```

Comment:
This isn't really what sensitive mode is for, this is more what the `_mas_pm_cares_about_dokis` var is for. However, I feel that you don't actually need that here. The phrasing here isn't really making fun of them, so it's fine to bring them up.

Reply:
I kept it anyway as if the player has indicated he doesn't care about the Doki's then Monika just won't bother mentioning them.

Reply:
That's not the main use for the var. It's whether or not the player cares about them being *insulted* or not.